### PR TITLE
[CFEP-14] security and systems subteam

### PR DESCRIPTION
**Pull request by beckermr**
Hi @conda-forge/core! 

Base on our discussion at the last meeting, I have put together this CFEP for a new subteam focused only security and our systems. 

I am posting a draft CFEP here for discussion. I think we are a ways off from a vote on this, but it is good to start a discussion!

### CODE REVIEW

File: cfep-14.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The core responsibilities of this team are to
+
+- maintain secure access to the credentials to all bot accounts
+- maintain secure access to all API keys used by `conda-forge`
+- maintain secure access to all third-party provider accounts owned by `conda-forge`
+- maintain the infrastructure for CFEP-13
+- develop automated ways to recover and reprovision `conda-forge` systems
+- perform ongoing security maintenance tasks like rotating API keys
+
```
Comment (CJ-Wright) [inline cfep-14.md]: Should this also have a reporting to core function?

Comment (beckermr) [reply cfep-14.md]: Yes agreed. As this team is only composed of core members, my idea is that this is really a subset of core that is focused on these tasks so they don’t fall through the cracks so to speak.

Comment (beckermr) [reply cfep-14.md]: I pushed an edit. LMK!

**Review by scopatz — APPROVED** [review]

**Review by msarahan — APPROVED** [review]

**Review by mwcraig — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Review by jakirkham — APPROVED** [review]

**Review by ericdill — APPROVED** [review]

**Review by jjhelmus — APPROVED** [review]

**Comment by beckermr** [comment]
@conda-forge/core

This PR falls under the sub-team formation policy, please vote and/or comment on this PR.
This PR needs 50% of core to vote yea to pass.
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch.
This vote will end on 2020-03-24.

**Review by pkgw — APPROVED** [review]

**Review by ocefpaf — APPROVED** [review]

**Review by patricksnape — APPROVED** [review]

**Review by djsutherland — APPROVED** [review]

**Review by mariusvniekerk — APPROVED** [review]

**Review by mbargull — APPROVED** [review]

**Comment by beckermr** [comment]
This passed! Merging. Vore summary PR is here: https://github.com/conda-forge/conda-forge.github.io/pull/1020

**Pull request merged**